openpype/plugins/publish/collect_frame_range_data.py

```diff
@@ -0,0 +1,85 @@
+import pyblish.api
+import clique
+from openpype.pipeline import publish
+from openpype.lib import BoolDef
+from openpype.lib.transcoding import IMAGE_EXTENSIONS
+
+
+class CollectFrameRangeData(pyblish.api.InstancePlugin,
+                            publish.OpenPypePyblishPluginMixin):
+    """Collect Frame Range data.
+    """
+
+    order = pyblish.api.CollectorOrder + 0.491
+    label = "Collect Frame Range Data"
+    families = ["plate", "pointcache",
+                "vdbcache", "online",
+                "render"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        repres = instance.data.get("representations")
+        asset_data = None
+        if repres:
+            first_repre = repres[0]
+            ext = ".{}".format(first_repre["ext"])
+            if "ext" not in first_repre:
+                self.log.warning(f"Cannot find file extension"
+                                 " in representation data")
+                return
+            if ext not in IMAGE_EXTENSIONS:
+                self.log.info("Collecting frame range data"
+                              " only supported for image extensions")
+                return
+
+            files = first_repre["files"]
+            repres_files, remainder = clique.assemble(files)
+            repres_frames = list()
+            for repres_file in repres_files:
+                repres_frames = list(repres_file.indexes)
```

Comment:
```suggestion
            collections, remainder = clique.assemble(files)
            if not collections:
                # No sequences detected so we can't retrieve
                # frame range
                return
                
            # Let's assume we only ever find a single collection
            # or we should error out otherwise? (or take min-max of ranges?)
            collection = collections[0]
            repre_frames = list(collection.indexes)
```